src/components/application_manager/include/application_manager/commands/command_impl.h

```diff
@@ -138,6 +138,14 @@ class CommandImpl : public Command {
   */
   void SetAllowedToTerminate(const bool allowed) OVERRIDE;
 
+  /**
+   * @brief Check syntax of string from mobile
+   * @param str - string that need to be checked
+   * @param allow_empty_string if true methods allow empty sting
+   * @return true if success otherwise return false
+   */
+  bool CheckSyntax(const std::string& str, bool allow_empty_line = false) const;
```

Comment:
@ZhdanovP propose to move this function to MessageHelper, make it static. 
And make optional argument as enum, but not Boolean. Also all couple of unit tests explicitly for this function

Reply:
@LuxoftAKutsan I think now the CheckSyntax is convenient for use. Almost all commands use CheckSyntax internal.
Moreover If we will move it to MessageHelper, a lot of files with changes will be. I think this changes will be better to provide in separate PR if you insist on this changes.

Reply:
@ZhdanovP  ok , but any way please check unit tests for it